### PR TITLE
[Poro] Using Materials.json

### PR DESCRIPTION
**Pull request by ipouplana**
Summary:
- The elements' materials will be defined from a materials.json file.
- The conditions' materials will still be read from the mdpa.
- Other minor fixes related

### CODE REVIEW

File: applications/PoromechanicsApplication/tests/element_tests/undrained_soil_column_2D/undrained_soil_column_2D.mdpa [hunk 1]
```diff
@@ -5,6 +5,8 @@ Begin Table 1 TIME FACE_LOAD_Y
 End Table
 
 
+Begin Properties 0
+End Properties
 Begin Properties 1
```
Comment (loumalouomega) [inline applications/PoromechanicsApplication/tests/element_tests/undrained_soil_column_2D/undrained_soil_column_2D.mdpa:10]: You can remove these properties if not used

**Review by loumalouomega — APPROVED** [review]
Look my comment

File: applications/PoromechanicsApplication/tests/element_tests/undrained_soil_column_2D/undrained_soil_column_2D.mdpa (hunk 1 above)
Comment (ipouplana) [reply applications/PoromechanicsApplication/tests/element_tests/undrained_soil_column_2D/undrained_soil_column_2D.mdpa:10]: I am actually using them for the Conditions (look at the description of the PR). 
The thing is that my “simple” problemtype is prepared to obtain the properties of the faces of the mesh (conditions) from the elements making it easy to write the properties without the need of any specific submodelpart... If I had to write those condition properties associated to a submodelpart, I would need to create that submodelpart first...

**Comment by loumalouomega** [comment]
OK, as you see I approved the PR

**Pull request merged**